ui/src/reducers/ui.js

```diff
@@ -1,15 +1,21 @@
 import { fromJS } from 'immutable';
 
-import { UI_CLOSE_BANNER } from '../actions/actionTypes';
+import {
+  UI_CLOSE_BANNER,
+  UI_CHANGE_GUIDE_MODAL_VISIBILITY,
+} from '../actions/actionTypes';
 
 export const initialState = fromJS({
   closedBannersById: {},
+  guideModalVisibility: null,
```

Comment:
why not just ``true`` ``false``

Reply:
null has a meaning, it means it was never displayed before.